src/core/messages.ts

```diff
@@ -514,27 +581,41 @@ export class DefaultMessages
     this._listenerSubscriptionPoints.clear();
   }
 
+  // This function is used to convert the message action from Ably to the chat MessageEvents.
+  private _messageActionToMessageEvent(action: ChatMessageActions): MessageEvents | undefined {
```

Comment:
Does this need to be a method, couldn't it just be a map?

Reply:
I'll convert it to a map :)